flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskCopyFrontendFiles.java

```diff
@@ -108,12 +124,46 @@ public void execute() {
         long ms = (System.nanoTime() - start) / 1000000;
         log().info("Visited {} resources. Took {} ms.",
                 resourceLocations.size(), ms);
+
+        calculateHashForPackagedThemes(targetDirectory);
+    }
+
+    private void calculateHashForPackagedThemes(File targetDirectory) {
```

Comment:
```suggestion
    private void calculateHashesForThemeJson(File pacakgedThemesDirectory) {
```
Update name to reflect what is hashed and from where.

Reply:
Not needed anymore, because this method has been removed.